Docs/Modules/StickerMessages.md

```diff
@@ -0,0 +1,43 @@
+## Installation
```

Comment:
This file has been deleted in the latest version. Why is it included in this pull request? Could you make sure that you are using the latest version of the code?

Reply:
Deleted